src/test/java/fi/aalto/cs/intellij/actions/EnablePluginsActionTest.java

```diff
@@ -0,0 +1,46 @@
+package fi.aalto.cs.intellij.actions;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import com.intellij.ide.plugins.IdeaPluginDescriptor;
+import com.intellij.ide.plugins.IdeaPluginDescriptorImpl;
+import com.intellij.notification.Notification;
+import com.intellij.openapi.actionSystem.AnActionEvent;
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import org.jdom.JDOMException;
+import org.jetbrains.annotations.NotNull;
+import org.junit.Test;
+import org.mockito.Mockito;
+
+public class EnablePluginsActionTest {
+
+  @Test
+  public void testActionPerformed() throws IOException, JDOMException {
+    IdeaPluginDescriptorImpl ideaPluginDescriptor = getIdeaPluginDescriptor();
+    ideaPluginDescriptor.setEnabled(false);
+    List<IdeaPluginDescriptor> descriptors = new ArrayList<>();
+    descriptors.add(ideaPluginDescriptor);
+
+    assertFalse(ideaPluginDescriptor.isEnabled());
+
+    EnablePluginsAction enableMissingPluginsAction = new EnablePluginsAction("",
```

Comment:
and now it actually fails (as supposed to), because of the PluginManger is getting a `null` [here](https://github.com/Aalto-LeTech/intellij-plugin/pull/50/files#diff-317038de384a7f0c8773d8af5b76827cR30-R32). So it either should be mocked too, or, some base class of the IDE used ([like this one](https://github.com/JetBrains/intellij-community/blob/7862cef1afd5f574d3f6e4a4b1d6bf372c1a71f0/platform/testFramework/src/com/intellij/testFramework/fixtures/BasePlatformTestCase.java)) which I'm to try next